lib/engine/game/g_1849/game.rb

```diff
@@ -531,13 +578,25 @@ def check_distance(route, _visits)
         end
 
         def check_other(route)
+          check_e_train_track_type(route)
           return unless (route.stops.map(&:hex).map(&:id) & PORT_HEXES).any?
 
           raise GameError, 'Route must include two non-port stops.' unless route.stops.size > 2
         end
 
+        def check_e_train_track_type(route)
+          train = route.train
+          paths = route.paths
+
+          raise GameError, 'E-Trains cannot use narrow gauge' if train.name == 'E' && paths.any? { |p| p.track == :narrow }
```

Comment:
Is this message too long for the route display? If so, shorten to `cannot use narrow gauge`

Reply:
Looks ok to me: 
![image](https://github.com/user-attachments/assets/8d989ffb-d982-48bc-9d8f-4c25571a3e84)